### ISSUE TITLE
no-unsafe-references false positive from TypeScript types

### ISSUE THREAD

**Issue by joshkel** [playwright-community/eslint-plugin-playwright#214]
The following code triggers a `no-unsafe-references` warning starting in 1.1.x:

```ts
import type { Store } from 'redux';

  async dispatch(action: any) {
    return this.page.evaluate((action) => {
      const store = (window as any).__redux_store as Store;
      store.dispatch(action);
    }, action);
  }
```

Since this is only a typecast / type reference, it doesn't exist at runtime, and the plugin should not raise a warning.
